### PR TITLE
Update token

### PR DESCRIPTION
**Pull request by LydiaLane**

### CODE REVIEW

**Comment by cryptomanz1** [comment]
Hello, can you send the transaction hash for the payment?

**Comment by cryptomanz1** [comment]
• What is the purpose of this token? How will it benefit Trust Wallet users?
 • How is the token being distributed? Are you conducting an airdrop?
 • Did you sold the tokens at an Initial Coin Offering (ICO) or a token presale?
 • Is the token listed on a centralized/decentralized exchange?
 • Is a Smart Contract audit completed? Where can we find this information?

File: blockchains/smartchain/assets/0x27Ae27110350B98d564b9A3eeD31bAeBc82d878d/info.json [hunk 1]
```diff
@@ -1,15 +1,14 @@
 {
     "name": "CumRocket",
     "website": "https://cumrocket.io",
-    "description": "CumRocket (CUMMIES) is a deflationary token used for purchasing adult content on their NSFW platform. Users are able to buy, sell and collect 18+ NFTs from their favourite models, as well as chat, tip and subscribe to exclusive content. 5% tax on every transaction - 2.5% burn, 2.5% redistribution.",
+    "description": "💦🚀",
```
[the remark this answers is not in the document]
Comment (LydiaLane) [reply blockchains/smartchain/assets/0x27Ae27110350B98d564b9A3eeD31bAeBc82d878d/info.json]: ok thanks, updated!

**Review by LydiaLane — COMMENTED** [review]
Updated

**Pull request merged**